src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangle extends Figure {
+    private String name = "Right triangle";
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
```

Comment:
I think you see that the logic of determining random values is repeated in every figure.